ledfx/effects/random.py

```
@@ -0,0 +1,92 @@
+import random
+import time
```

Ruff

`time` imported but unused

Remove unused import: `time`

(F401)